src/vscode-bicep/src/panes/deploy/app/components/index.css

```diff
@@ -0,0 +1,29 @@
+#webview-body {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-section {
+  display: grid;
+  grid-auto-flow: row;
+  grid-row-gap: 1rem;
+  overflow-x: scroll;
+}
+
+.input-row {
+  display: grid;
+  grid-auto-flow: row;
+}
+
+@media screen and (min-width: 1024px) {
+  #webview-body {
+    padding: 0 8rem;
+    max-width: 1024px;
+    margin: 0 auto;
+  }
+}
+
+vscode-button {
+  width: fit-content;
+  margin-right: 0.5rem;
+}
```

Comment:
non-blocking: I think we can set css with styled-components instead, since it's already a dependency of the VS Code extension.